pages/common/tmsu.md

```diff
@@ -0,0 +1,28 @@
+# tmsu
+
+> Simple command-line tool for tagging files.
+> More information: <https://tmsu.org>.
+
+- Tag a specific file with multiple tags:
+
+`tmsu tag {{summer.mp3}} {{music}} {{big-jazz}} {{mp3}}`
+
+- Tag multiple files:
+
+`tmsu tag --tags {{"music mp3"}} {{*.mp3}}`
+
+- List tags of specified file(s):
+
+`tmsu tags {{*.mp3}}`
+
+- List files with specified tag(s):
+
+`tmsu files {{big-jazz}} {{music}}`
+
+- List files with tags matching boolean expression:
+
+`tmsu files "{{(year >= 1990 and year <= 2000)}} and {{grunge}}"`
+
+- Mount tmsu virtual filesystem to an existing directory:
+
+`tmsu mount {{path/to/tmsu-virtual-fs}}`
```

Comment:
We use `snake_case` for placeholders, but this is a path to a directory, I think this is better

```suggestion
`tmsu mount {{path/to/directory}}`
```